### PR TITLE
WIP: Replace plotly HTML rendering with Panel

### PR DESCRIPTION
**Pull request by jonmmease**
Rough initial prototype of using Panel to display plotly figures in HoloViews.  The goal is to combine this with https://github.com/pyviz/holoviews/pull/3836 and https://github.com/pyviz/panel/pull/529 to support interactive streams with the Plotly backend.

If successful, this would become the new foundation for the work started in https://github.com/pyviz/holoviews/pull/3316 to support `Selection1D` and `RangeXY` streams with the Plotly backend.

### CODE REVIEW

File: holoviews/plotting/plotly/__init__.py [hunk 1]
```diff
@@ -16,13 +16,17 @@
...
+
 
 if LooseVersion(plotly.__version__) < '3.4.0':
     raise VersionError(
         "The plotly extension requires a plotly version >=3.4.0, "
         "please upgrade from plotly %s to a more recent version."
         % plotly.__version__, plotly.__version__, '3.4.0')
 
+pn.extension("plotly")
+
```
Comment (philippjfr) [inline holoviews/plotting/plotly/__init__.py]: Should remove this, importing the plotting module shouldn't load the extension.

File: holoviews/plotting/plotly/__init__.py [hunk 2]
```diff
@@ -16,13 +16,17 @@
 from .tabular import *               # noqa (API import)
 from ...core.util import LooseVersion, VersionError
 import plotly
+import panel as pn
```
Comment (philippjfr) [inline holoviews/plotting/plotly/__init__.py]:
```suggestion
import panel.models.plotly # noqa
```

**Comment by philippjfr** [comment]
Looks like a good start. Will probably have to add something to either the extension itself or `PlotlyRenderer.load_nb` to load the appropriate parts of the extension.

**Comment by jonmmease** [comment]
Superseded by https://github.com/pyviz/holoviews/pull/3850

**Pull request closed without merging**